airflow/models/dag.py

```diff
@@ -1538,7 +1538,7 @@ def pickle_info(self):
             dttm = timezone.utcnow()
             pickled = pickle.dumps(self)
             d['pickle_len'] = len(pickled)
-            d['pickling_duration'] = "{}".format(timezone.utcnow() - dttm)
+            d['pickling_duration'] = f"{timezone.utcnow() - dttm}"
```

Comment:
Here using `str(timezone.utcnow() - dttm})` may make more sense than f"{timezone.utcnow() - dttm}" IMO, given it's purely a type conversion rather than formating.